packages/coinstac-client-core/src/sub-api/project-service.js

```diff
@@ -66,8 +66,26 @@ class ProjectService extends ModelService {
           this.dbRegistry.get('consortia').get(project.consortiumId),
         ]);
       })
-      .then(([project, consortium]) => {
-        const covariates = consortium.activeComputationInputs[0][2];
+      .then(([project, consortium]) => Promise.all([
+        project,
+        consortium,
+        this.dbRegistry.get('computations').get(consortium.activeComputationId),
+      ]))
+      .then(([project, consortium, computation]) => {
+        const covariatesIndex = computation.inputs[0].findIndex(({ type }) => {
```

Comment:
We're still requiring a “covariates” input here. It’s fine because both multi-shot and single-shot have this input, but this will have to be improved in the future.